src/common/auth.js

```diff
@@ -0,0 +1,62 @@
+import Bell from "@hapi/bell";
+import Cookie from "@hapi/cookie";
+import { config } from "./config.js";
+
+export const validateSession = (_request, session) => {
+  if (!session || !session.profile) {
+    return { isValid: false };
+  }
+
+  return { isValid: true, credentials: session };
+};
+
+export const auth = {
+  plugin: {
+    name: "auth",
+    once: true,
+    async register(server) {
+      await server.register([Bell, Cookie]);
+
+      server.auth.strategy("session", "cookie", {
+        cookie: {
+          name: "session-auth",
+          password: config.get("session.cookie.password"),
```

Comment:
`SESSION_COOKIE_PASSWORD` is not defined in CDP, so the default value that has been checked into the repo will be used. We should store a new one in secrets.